modules/nodes-pods-autoscaling-creating-memory.adoc

```diff
@@ -169,7 +169,7 @@ horizontalpodautoscaler.autoscaling/hpa-resource-metrics-memory created
 $ oc get hpa hpa-resource-metrics-memory
 
 NAME                                     REFERENCE                       TARGETS         MINPODS   MAXPODS   REPLICAS   AGE
-oc get hpa hpa-resource-metrics-memory   ReplicationController/example   2441216/500Mi   1         10        1          20m
+oc get hpa hpa-resource-metrics-memory   ReplicationController/example   2441216/500m   1         10        1          20m
```

Comment:
This one we want to leave at `500Mi`, but like the CPU one, the name column shouldn't have `oc get` in it, just the name `hpa-resource-metrics-memory`